Dockerfile

```diff
@@ -12,15 +12,14 @@ ADD setup.py /opt/atomicapp/
 ADD requirements.txt /opt/atomicapp/
 
 WORKDIR /opt/atomicapp
-
-RUN pip install .
+RUN python setup.py install
```

Comment:
`pip install` finally reads and installs dependencies from the list passed to `install_requires` of method `setup`, and other dependency related arguments. It does not install dependencies from [Requirement Files](https://pip.pypa.io/en/stable/user_guide.html#requirements-files) automatically.